dashboard/models.py

```diff
@@ -84,8 +84,8 @@ def set_user_defaults(self, course_id, user_id, default_view_type, default_view_
 
 class UserDefaultSelection(models.Model):
     id = models.AutoField(primary_key=True, verbose_name="Table Id")
-    course_id = models.CharField(max_length=255, blank=True, null=True, verbose_name="Course Id")
-    user_id = models.CharField(max_length=255, blank=True, null=True, verbose_name="User Id")
+    course_id = models.BigIntegerField(blank=True, null=True, verbose_name="Course Id")
+    user_id = models.BigIntegerField(blank=True, null=True, verbose_name="User Id")
```

Comment:
The user default piece is broken due to this change. the `user_id` is actually the sis_user_name not the user_id. So this still needs to be `CharFiled`. due to the confusion in the naming convention, I felt the column name needs to be changed. Then it will be a new migration file will be generated something like this 
```
class Migration(migrations.Migration):

    dependencies = [
        ('dashboard', '0005_auto_20190613_1502'),
    ]

    operations = [
        migrations.AddField(
            model_name='userdefaultselection',
            name='user_sis_name',
            field=models.CharField(blank=True, max_length=255, null=True, verbose_name='User SIS name'),
        ),
        migrations.RemoveField(
            model_name='userdefaultselection',
            name='user_id',
        ),
        migrations.AlterUniqueTogether(
            name='userdefaultselection',
            unique_together=set([('user_sis_name', 'course_id', 'default_view_type')]),
        ),
    ]
```
 I can commit both the files change and the new migration change if you want me to.